### DISCUSSION TITLE
Compatibility with Dropbox short-lived access tokens

### DISCUSSION THREAD

**Discussion by cashewcodes9** [spatie/flysystem-dropbox#78] [Q&A]
Hello Great human beings,

As Dropbox introduced Scopes, PKCE, short-lived access token and refresh tokens for authorization. The long-lived access token will be depreciated on September 30th 2021. I really liked `spatie/flyszstem-dropbox` package but I want to ask the community if it is compatible with the new short-lived access token and refresh token. 

https://dropbox.tech/developers/migrating-app-permissions-and-access-tokens

My usecase will be to store reports generated by my laravel application to dropbox. And very much similar to this blog post: https://nemanjacoder.medium.com/using-dropbox-as-storage-for-laravel-f6c401b97663

I need to make decision this week so would really appreciate a feedback. Thank you.

**Comment by cashewcodes9** [comment]
Thanks, Answered by this package.
https://github.com/spatie/dropbox-api

**Reply by Ojsholly** [reply]
Please how were you able to implement the dropbox-api package with the flysystem-dropbox package? I'm currently facing the same issue of short-lived access tokens right now.
